packages/manager/modules/manager-components/src/task-tracker/task-tracker.controller.js

```diff
@@ -1,23 +1,18 @@
 import { POLL_INTERVAL, TASK_STATUS } from './task-tracker.constants';
 
-let uid = 0;
-
 export default class TaskTrackerController {
   /** @ngInject */
```

Comment:
should be 
```suggestion
  /* @ngInject */
``` ? 
I'm not sure if both syntax are valid

Reply:
:+1: 

Looks like it works tho
```javascript
TaskTrackerController.$inject = ["$translate", "$timeout", "$http", "TaskTrackerService"];
function TaskTrackerController($translate, $timeout, $http, TaskTrackerService) {
```